code/src/UI/Services/ProjectConfigInfoService.cs

```diff
@@ -443,9 +477,23 @@ private static bool ContainsNugetPackage(string packageId)
             return false;
         }
 
-        private static bool ContainsSDK(string sdkId)
+        private bool ContainsNugetPackageCpp(string packageId)
```

Comment:
Why this name? It doesn't appear to have anything specific to C++/CPP in it.

Reply:
We look for added nuget in C# (csproj) and C++ (packages.config) in different files

Reply:
@mvegaca,  Can we call this ContainsNugetPackageInPackageConfig or do a method that's called ContainsNugetPackage and looks in both *.proj file and packages.config file? packages.config is not something specific to C++

Reply:
Changed in https://github.com/microsoft/WindowsTemplateStudio/pull/4114/commits/8dfb2d94542631a98f8e69addbd63ed3711f4e1c after talking to @mvegaca